common/modal.js

```diff
@@ -17,20 +17,27 @@
             $uibModalInstance.dismiss('cancel');
         }
     }])
-    .controller('ErrorModalController', ['$uibModalInstance', 'params', 'messageMap', function ErrorModalController($uibModalInstance, params, messageMap) {
+    .controller('ErrorModalController', ['$uibModalInstance', 'params', 'messageMap', '$window', function ErrorModalController($uibModalInstance, params, messageMap, $window) {
         var vm = this;
         vm.params = params;
         vm.details = false;
         vm.linkText = messageMap.showErrDetails;
+        vm.reloadBtn = false;
+        var reloadMessage = ' <p>  </p>';
 
         if(vm.params.errorCode == 'Multiple Records Found'){
-          vm.clickActionMessage =  messageMap.recordAvailabilityError.multipleRecords;
+            vm.clickActionMessage =  messageMap.recordAvailabilityError.multipleRecords;
         } else if(vm.params.errorCode == 'Record Not Found'){
-          vm.clickActionMessage = messageMap.recordAvailabilityError.noRecordsFound;
+            vm.clickActionMessage = messageMap.recordAvailabilityError.noRecordsFound;
         } else {
-          vm.clickActionMessage = messageMap.recordAvailabilityError.pageRedirect + vm.params.pageName;
+            vm.clickActionMessage = messageMap.recordAvailabilityError.pageRedirect + vm.params.pageName + '. ';
+            vm.reloadBtn = false;
+            if(vm.params.appName == 'recordedit'){
+              vm.reloadBtn = true;
+              reloadMessage = ' <p>' + messageMap.terminalError.reloadMessage +' </p>';
+            }
         }
-
+        vm.clickActionMessage += reloadMessage;
```

Comment:
Is the intention for the `clickActionMessage` to be before an empty `<p>` tag? Do we need the `reloadMessage` if there is no content inside the `<p>` tag?

Reply:
`<p>` tag is added to maintain the space between click action message and buttons. Also maintains consistency in their placement irrespective of reload message